src/resources/Microsoft.Authorization/policySetDefinitions/Deploy-MDFC-Config.json

```
@@ -8,7 +8,7 @@
     "displayName": "Deploy Microsoft Defender for Cloud configuration",
     "description": "Deploy Microsoft Defender for Cloud configuration",
     "metadata": {
-      "version": "3.1.1",
+      "version": "4.1.1",
```

```suggestion
      "version": "4.0.0",
```

When you increase a major/minor, you always reset the others to 0 👍